### PR TITLE
Factorize font path computation

### PR DESCRIPTION
**Pull request by avalentino**
This PR does not fix any issue and do not provide any new feature.
It simply factorize the computation of the `DejaVuSerif.ttf` font path in a single point and uses the computed variable where needed.
It is a purely aesthetic change, but it makes life a little bit easier to packager that need to patch the font path.

Please note that test failures are unrelated to this change.
They are due to an incompatibility with Pillow v9.4 described in #82.

 - [ ] Tests passed 
 - [X] Passes ``git diff origin/main **/*py | flake8 --diff``

### CODE REVIEW

**Review by djhoese — APPROVED** [review]
Nice job! Thanks. Let me take a look at the test failures and see if I can come up with a quick fix.

**Review by mraspaud — APPROVED** [review]
LGTM

**Comment by djhoese** [comment]
I merged @mraspaud's PR into main and tried merging it into your branch. I think I made all the necessary fixes for the conflicts...oops pre-commit is failing. I guess not.

File: pycoast/tests/test_pycoast.py [hunk 1]
```diff
@@ -351,7 +352,7 @@ def test_grid_germ(self, cw_pil):
         area_def = GERM
 
         cw_pil.add_coastlines(img, area_def, resolution="l", level=4)
-        font = ImageFont.truetype(os.path.join(os.path.dirname(__file__), "test_data", "DejaVuSerif.ttf"), 16)
+        font = ImageFont.truetype(image_path, 16)
```
Comment (djhoese) [inline pycoast/tests/test_pycoast.py:355]:
```suggestion
        font = ImageFont.truetype(font_path, 16)
```

**Comment by djhoese** [comment]
@avalentino Do you need a release with these recent changes (pillow 9.4 and font path) to make your life easier?

**Comment by avalentino** [comment]
> @avalentino Do you need a release with these recent changes (pillow 9.4 and font path) to make your life easier?

If I understand correctly the actual code was already OK, only the test code have been changes since the last release.
For the test code I already have a workaround in place so a new delivery is not urgent for me.
When you will make a new release I will remove debian specific patches to workaround test issues.

Thanks a lot @djhoese and @mraspaud

**Pull request merged**